weather_get.py

```diff
@@ -0,0 +1,41 @@
+import sys
+import os.path
+sys.path.append(os.path.dirname(__file__))
+import requests
+import json
+
+from config import WEATHER_API_KEY
+one_call_base = 'https://api.openweathermap.org/data/2.5/onecall?'
+path = os.path.join(os.path.dirname(__file__), '..', 'routes\ASC2021\ASC2021_draft.csv')
+with open(path, 'r') as wea2021:
+    for line in wea2021:
+        # read a set of latitude and longitude points
+        row = line.split(',')
+        location = [ ]
+        for i in range (len(row)):
+            i = int(i)
+            lat = row[i][0].strip()
+            lon = row[i][1].strip()
+            location.append(lat)
+            location.append(lon)
+            n = 2
+            locations = [location[j * n:(j + 1) * n] for j in range((len(location) + n - 1) // n )]  
+        #for point #1
+        lat = locations[0][0].strip()
+        lon = locations[0][1].strip()

```

Comment:
I'm a tad bit confused around what's happening here? Assuming that from the CSV, we want a list like:

`[[lat, lon], [lat, lon], [lat, lon]]`

on which we're going to make API calls for each point?

Reply:
In my head, the easiest way to do that is to declare an empty list before line 11, and for each row, use `.append([lat, lon])`